packages/postcss-ordered-values/src/rules/listStyle.js

```diff
@@ -0,0 +1,45 @@
+import valueParser from 'postcss-value-parser';
+
+const definedTypes = [
+  'disc',
+  'circle',
+  'square',
+  'decimal',
+  'georgian',
+  'trad-chinese-informal',
+  'kannada',
+];
```

Comment:
I think in this case we better not store all the values, in the future there may be more of them, is it possible to rewrite it?

```if
if (definedPosition.includes(decl.value)) {
} else if (decl.value === 'none')
else if (/* Something */) {
} else {
  // its a type field
  order.type = `${order.type} ${decl.value}`;
}
```

Reply:
may be need to fetch them from here. https://github.com/mdn/browser-compat-data/blob/master/css/properties/list-style-type.json

Reply:
done